docs/docsite/rst/scenario_guides/vmware_scenarios/vmware_inventory_hostnames.rst

```diff
@@ -0,0 +1,125 @@
+.. _vmware_ansible_inventory_using_hostnames:
+
+*************************************************
+Using VMware dynamic inventory plugin - Hostnames
+*************************************************
+
+.. contents:: Topics
+
+VMware Dynamic Inventory Plugin - Customizing hostnames
+=======================================================
+
+
+VMware inventory plugin allows to configure hostnames using ``hostnames`` configuration parameter.
+
+In this scenario guide we will see how you configure hostnames from the given VMware guest in the inventory.
+
+Requirements
+------------
+
+To use the VMware dynamic inventory plugins, you must install `pyVmomi <https://github.com/vmware/pyvmomi>`_
+on your control node (the host running Ansible).
+
+To include tag-related information for the virtual machines in your dynamic inventory, you also need the `vSphere Automation SDK <https://code.vmware.com/web/sdk/65/vsphere-automation-python>`_, which supports REST API features like tagging and content libraries, on your control node.
+You can install the ``vSphere Automation SDK`` following `these instructions <https://github.com/vmware/vsphere-automation-sdk-python#installing-required-python-packages>`_.
+
+.. code-block:: bash
+
+    $ pip install pyvmomi
+
+VMware dynamic inventory plugin is now available in the VMware Collection, so you need to install VMware collection via -
+
+.. code-block:: bash
+
+    $ ansible-galaxy collection install community.vmware
+
```

Comment:
since community.vmare is part of Ansible 2.10, you don't need the separate install above.  If you feel it's important to include this, then add a sentence above it to say "To install the latest ``community.vmare`` collection:"